src/webapp/pages/home/HomePage.tsx

```diff
@@ -215,7 +217,8 @@ const AdditionalComponents: React.FC<{
 };
 
 export const HomePage: React.FC = React.memo(() => {
```

Comment:
This file is pretty big. I would extract the different components `if() return <></>` and the AdditionalComponents component into each own file.